### ISSUE TITLE
Distinguish between canals and natural waterways

### ISSUE THREAD

**Issue by 1ec5** [ZeLonewolf/openstreetmap-americana#760]
Canals look just like natural waterways, which can be confusing whenever both a canal and a creek is in view:

[<img src="https://user-images.githubusercontent.com/1231218/216527800-aebc74fc-3625-41e0-8df4-26cb556411f8.png" width="400" alt="Metamora">](https://zelonewolf.github.io/openstreetmap-americana/#map=13/39.44688/-85.1381)

Even worse is when a canal follows an aqueduct over a creek:

[<img src="https://user-images.githubusercontent.com/1231218/216522744-0f4de0bf-988f-490e-9313-a42b55102e78.png" width="400" alt="Aqueduct">](https://zelonewolf.github.io/openstreetmap-americana/#map=17/39.446273/-85.130071)

One approach I’ve seen in print maps is to depict a canal as a line with teeth, which are probably intended to evoke locks at a regular interval, even if the locks themselves are omitted:

<img width="260" alt="CRAM Political World" src="https://user-images.githubusercontent.com/1231218/216523060-efd72437-7af4-4c47-a918-55ae87efe9d6.png">

**Comment by jleedev** [comment]
Bridge casing would help, of course:

![image](https://user-images.githubusercontent.com/23022/216601759-06082ccf-fd18-4c50-9c52-c9c8be73ac2f.png)

**Comment by adamfranco** [comment]
> One approach I’ve seen in print maps is to depict a canal as a line with teeth, which are probably intended to evoke locks at a regular interval, even if the locks themselves are omitted

I wonder if those are meant to invoke a cutting or embankment? We could add a mid-tone casing or some other treatment to act as sort of a shadow line for now, then unify with embankments/cuttings if we ever add them.

**Comment by ZeLonewolf** [comment]
Is this ticket specifically about canals that are represented as linear features? (i.e. a `waterway=canal` object but no `natural=water` + `water=canal` area)?

**Comment by jleedev** [comment]
Probably wouldn't make sense for those that have an area:

[<img src="https://user-images.githubusercontent.com/23022/216635407-7820d32b-4ce7-4a57-a539-3567f807623f.png" alt="Genesee River / Erie Canal">][1]

[1]: https://zelonewolf.github.io/openstreetmap-americana/#map=16/43.12065/-77.641745

**Comment by 1ec5** [comment]
> Is this ticket specifically about canals that are represented as linear features? (i.e. a `waterway=canal` object but no `natural=water` + `water=canal` area)?

Yes. Despite my second screenshot above, I’m mostly thinking of these canals at zoom levels where all you see is a linear feature. If you zoom in enough to where the area gets wide, then it’ll cover up both the line and any ornamentation, but I think that’s OK. At that point, we want things like rivers and roads to look more realistic and less symbolized.

> I wonder if those are meant to invoke a cutting or embankment? We could add a mid-tone casing or some other treatment to act as sort of a shadow line for now, then unify with embankments/cuttings if we ever add them.

I think this is unlikely, since the transportation maps I’ve seen this on don’t have any visual language for cuttings or embankments in general. Personally, I’m reminded of the canals whose converted towpaths I’ve biked along, where the remnants of these locks appear at a regular interval. But it could just be an arbitrary abstract symbol. It’s just that I’ve seen it in multiple unrelated maps, so I figured it’s something of a convention.

To be sure, not every map distinguishes canals this way. Many leave them looking just like natural waterways. This official tourism map of Arizona distinguishes canals by pointing out that they’re straighter than rivers, duh:

<img width="252" alt="Arizona Office of Tourism 2021" src="https://user-images.githubusercontent.com/1231218/216711224-cb91d678-8008-4c96-90ac-d64528ba2bcc.png">

**Comment by ZeLonewolf** [comment]
One technical challenge I see is that if we go with the "teeth" option, is that we would want to shift the label laterally as we do for streets.  This side effect would make the label look oddly offset when you _do_ have the canal water area visible.

I like the teeth concept though if we can get past this labeling issue.
